CHANGELOG.md

```diff
@@ -33,7 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Patcher Changes
 
-- Nothing.
+- Option to use deterministic I. Drone RNG for fairer racing
```

Comment:
could just use the full name?

Reply:
It's I. Drone in the Logbook XD

Reply:
Really? That'd be enough reasoning for me lol